yt/utilities/tests/test_parameter_file_storage.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import pytest
+
+import yt
+from yt.config import ytcfg
+from yt.utilities.parameter_file_storage import ParameterFileStore
+
+
+@pytest.fixture
+def store_parameter_files():
+    spf_init = ytcfg.get("yt", "store_parameter_files")
+    ytcfg.set("yt", "store_parameter_files", True)
+    yield
+    ytcfg.set("yt", "store_parameter_files", spf_init)
+
+
+@pytest.fixture
+def set_parameter_file(tmp_path):
+    pfs_init_name = ytcfg.get("yt", "parameter_file_store")
+    pfs_path = tmp_path / "param_file_store.csv"
+    pfs_path.touch()
+    ytcfg.set("yt", "parameter_file_store", str(pfs_path))
+    yield
+    ytcfg.set("yt", "parameter_file_store", pfs_init_name)
+
+
+def test_on_disk_parameter_file_store(
+    monkeypatch, store_parameter_files, set_parameter_file
```

Comment:
Fixtures that are used as setup/teardown but don't need an accessible name should be declared with `pytest.mark.usefixture`